application-analytics-ui/src/main/resources/Analytics/Code/Macros/MostViewedPages.xml

```diff
@@ -241,31 +241,37 @@
     </property>
     <property>
       <code>{{include reference='Analytics.Code.Macros.VelocityMacros'/}}
+
 {{velocity}}
 #getDate()
 #set ($period = 'range')
 #set ($parameters = $escapetool.url({
+  'resultPage': 'Analytics.Code.Macros.MostViewedPagesJSON',
   'date': $date,
   'period': $period,
-  'limitEntries': '-1'
+  'limitEntries': '-1',
+  'translationPrefix': 'analytics.mostViewedPages.header.'
 }))
-#set ($columns = ['date', 'title', 'visits', 'hits', 'timeSpent', 'entryNbActions', 'bounceRate', 'exitRate'])
-#set ($columnsProperties = {
-  'date' : { 'type' : 'text', 'sortable' : true },
-  'title' : { 'type' : 'text', 'sortable' : true },
-  'visits' : { 'type' : 'text', 'sortable' : true },
-  'hits' : { 'type' : 'text', 'sortable' : true },
-  'timeSpent' : { 'type' : 'text', 'sortable' : true },
-  'entryNbActions' : { 'type' : 'text', 'sortable' : true },
-  'bounceRate' : { 'type' : 'text', 'sortable' : true },
-  'exitRate' : { 'type' : 'text', 'sortable' : true }
-})
-#set($options = {
-  'translationPrefix': 'analytics.mostViewedPages.header.',
-  'resultPage': 'Analytics.Code.Macros.MostViewedPagesMacro.MostViewedPagesJSON',
-  'extraParams': $parameters
+#set ($liveDataConfig={
+  'meta':{
+    'propertyDescriptors': [
+      {'id': 'title', 'displayer': 'html'},
+      {'id': 'visits', 'displayer': 'html'},
+      {'id': 'hits', 'displayer': 'html'},
+      {'id': 'timeSpent', 'displayer': 'html'},
+      {'id': 'entryNbActions', 'displayer': 'html'},
+      {'id': 'bounceRate', 'displayer': 'html'},
+      {'id': 'exitRate', 'displayer': 'html'}
+    ]}
 })
-#livetable('mostViewedPages' $columns $columnsProperties $options)
+
+{{liveData
+  id="mostViwedMacro"
```

Comment:
Typo

Reply:
The change for having different ids (in case there are multiple macros on a page) is in a different PR, right? I wanted to be sure:)

Reply:
The change is in the RowEvolution pr